File: python/1043.py
a, b, c = input().split()
a = float(a)
b = float(b)
c = float(c)

perimeter = a + b + c
area = ((a + b) * c) / 2

if a + b > c and b + c > a and a + c > b:
    print(f'Perimetro = {perimeter:.1f}')
else:
    print(f'Area = {area:.1f}')

File: lua/1043.lua
local entry = io.read()
local t = {}

for v in string.gmatch(entry, "[^%s]+") do
    table.insert(t, tonumber(v))
end

local a = t[1]
local b = t[2]
local c = t[3]

if a + b > c and b + c > a and a + c > b then
    print('Perimetro = '..string.format('%.1f', a+b+c))
else
    print('Area = '..string.format('%.1f', ((a+b)*c)/2))
end
